packages/legacy-scripting-runner/index.js

```diff
@@ -680,6 +691,7 @@ const legacyScriptingRunner = (Zap, zcli, input) => {
 
       if (!options.parseResponse) {
         if (options.checkResponseStatus) {
+          response.skipThrowForStatus = false;
```

Comment:
Setting `skipThrowForStatus` to false before every `throwForStatus()` is only necessary for core 10+. Otherwise, `throwForStatus()` won't throw anything on core 10+.

core 9.4.0 doesn't have this `skipThrowForStatus` feature and will ignore.